tests/ConnectionTest.php

```diff
@@ -396,6 +396,46 @@ public function testConnectStartsTransactionInNoAutoCommitMode(): void
         self::assertTrue($conn->isTransactionActive());
     }
 
+    public function testTransactionIsNotActiveAfterTransactionalInAutoCommitMode(): void
+    {
+        $driverMock = $this->createStub(Driver::class);
+        $driverMock
+            ->method('connect')
+            ->willReturn(
+                $this->createMock(DriverConnection::class),
```

Comment:
You're not storing the result of this expression in a variable, therefore it should be a stub.